web_widget_timepicker/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Vividlab (<http://www.vividlab.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Web Timepicker Widget",
+    "version": "12.0.1.0.0",
+    "author": u"VividLab, "
```

Comment:
No need for any `u`'s. This is python 3.x and therefore strings are Unicode by definition.

Reply:
Done